src/containerUtils.js

```diff
@@ -0,0 +1,58 @@
+import invariant from 'invariant';
+import React from 'react';
+import Relay from 'react-relay';
+
+import batchedRelayContainerBase from './batchedRelayContainerBase';
+
+export function createBatchedRelayContainer(
+    children,
+    params,
+    name,
+    queryAggregator
+) {
+  const {queries, fragments} = queryAggregator.flush();
+  const BatchedRelayContainer = {fragments, ...batchedRelayContainerBase};
+  const route = {name, queries, params};
+  const render = () => children;
+
+  return (
+    <Relay.RootContainer
+      Component={BatchedRelayContainer}
+      route={route}
+      renderFailure={render}
+      renderLoading={render}
+      renderFetched={render}
```

Comment:
IMO the best default here would be to only specify `renderFetched`, and let the user specify `rootContainerProps` on the `Router`, for consistency with the below. The default `renderLoading` is pretty good for most use cases.